www/index.rst

```diff
@@ -28,44 +29,59 @@ few restrictions.
-- `NumPy Mailing List <https://mail.python.org/mailman/listinfo/numpy-discussion>`__
+To install NumPy, we strongly recommend using a
+*scientific Python distribution*.  See
+`Installing the SciPy Stack <https://www.scipy.org/install.html>`__ for
+details.
+
+Many high quality online tutorials, courses, and books are available to get
+started with NumPy. For a quick introduction to NumPy we provide the
+`NumPy Tutorial <https://www.numpy.org/devdocs/user/quickstart.html>`__. We
+also recommend the `SciPy Lecture Notes <https://scipy-lectures.org>`__ for a
+broader introduction to the scientific Python ecosystem.
 
 For more information on the SciPy Stack (for which NumPy provides the
 fundamental array data structure), see `scipy.org <https://www.scipy.org/>`__.
 
-
 Documentation
 -------------
 
-`Latest (development) version <https://www.numpy.org/devdocs>`__
-
-* `Version 1.16.1 <https://docs.scipy.org/doc/numpy-1.16.1>`__,
-  `[HTML+zip] <https://docs.scipy.org/doc/numpy-1.16.1/numpy-html-1.16.1.zip>`__
-  `[PDF] <https://docs.scipy.org/doc/numpy-1.16.1/numpy-ref-1.16.1.pdf>`__
-* `Version 1.15.4 <https://docs.scipy.org/doc/numpy-1.15.4>`__,
-  `[HTML+zip] <https://docs.scipy.org/doc/numpy-1.15.4/numpy-html-1.15.4.zip>`__
-  `[PDF] <https://docs.scipy.org/doc/numpy-1.15.4/numpy-ref-1.15.4.pdf>`__
-* `Version 1.14.5 <https://docs.scipy.org/doc/numpy-1.14.5>`__,
-  `[HTML+zip] <https://docs.scipy.org/doc/numpy-1.14.5/numpy-html-1.14.5.zip>`__,
-  `[PDF] <https://docs.scipy.org/doc/numpy-1.14.5/numpy-ref-1.14.5.pdf>`__
-* `Version 1.13.0 <https://docs.scipy.org/doc/numpy-1.13.0>`__,
-  `[HTML+zip] <https://docs.scipy.org/doc/numpy-1.13.0/numpy-html-1.13.0.zip>`__,
-  `[PDF] <https://docs.scipy.org/doc/numpy-1.13.0/numpy-ref-1.13.0.pdf>`__
+The most up-to-date NumPy documentation can be found at
+`Latest (development) version <https://www.numpy.org/devdocs>`__.  I
+It includes a user guide, full reference documentation, a developer guide, meta
+information, and "NumPy Enhancement Proposals" (which include the NumPy Roadmap
+and detailed plans for major new features).
 
-.. raw:: html
+A complete archive of documentation for all NumPy releases (minor versions; bug
+fix releases don't contain significant documentation changes) since 2009 can be
+found at https://docs.scipy.org. 
+
+Support NumPy
+-------------
 
+If you have found NumPy to be useful in your work, research or company, please
+consider making a donation to the project commensurate with your resources. Any
+amount helps! All donations will be used strictly to fund the development of
+NumPy’s open source software, documentation and community.
+
+Donations are managed by the NumFOCUS Foundation, which is the legal and fiscal
+umbrella for the project. NumFOCUS is a 501(c)3 non-profit foundation, so if
+you are subject to US Tax law, your contributions will be tax-deductible.
+NumPy's `Steering Council <https://www.numpy.org/devdocs/dev/governance/index.html>`__
```

Comment:
"will" -> "can"

Reply:
I think it's "will" - all contributions will be tax-deductable, as far as I know it's not conditional on anything.

Reply:
It is conditional on 
- the entity filing a tax return (some people may be subject to US Tax law and may not be filing returns)
- the return should itemize deductions and not take a standard deduction
- they need to list this donation in their deductions
- they need to receive and hang on to the receipt from NumFocus
- the IRS could audit them and disapprove the deduction, for reasons between the entity and the IRS, for instance if the IRS determines that the entity derived benefit from the donation. I am not a tax lawyer, I don't know all the conditions that may disallow deductions.

I would either leave off that whole part of the sentence, "NumFOCUS is a 501(c)3 non-profit foundation." or do something like [the Amrican Friends of the Technion site](https://secure2.convio.net/ats/site/Donation2?df_id=2302&mfc_pref=T&2302.donation=form1&s_src=website&s_subsrc=header_donate) "Contributions are tax deductible and no benefit was provided. Please consult your tax advisor." or [the American Cancer Society](https://donate3.cancer.org/?lang=en&_ga=2.241927488.314579350.1555067931-263619370.1555067931) "We are a qualified 501(c)(3) tax-exempt organization and donations are tax-deductible to the full extent of the law. No goods or services were provided for this gift. Please consult your tax advisor regarding specific questions about your deductions.

Reply:
I think you're looking for a problem where none exists. You are linking to actual donation forms with lots of detail. If you go to www.cancer.org (their front page) you get a popup with one sentence: "Your gift is 100% tax deductible". Not "can be" or "may be", but "is". 

And it's of course obvious that it requires asking for the deduction and following the relevant laws, like keeping receipts. Really that is all irrelevant for a front page.

Reply:
+1 for "is"

Reply:
okay changed to "is"